tests/serviceBuilder.test.js

```diff
@@ -1384,5 +1386,178 @@ tap.test('serviceBuilder', test => {
     assert.end()
   })
 
+  test.test('tls options', async assert => {
+    nock.enableNetConnect('localhost:3200')
+    assert.tearDown(() => {
+      nock.disableNetConnect()
+    })
+
+    async function createServer() {
+      return new Promise((resolve) => {
+        const server = https.createServer({
+          requestCert: true,
+          rejectUnauthorized: false,
+          ca: fs.readFileSync('tests/fixtures/keys/ca.crt'),
```

Comment:
can we disable these lint issues for these test cases? (or use the async version with await, if applicable)